src/components/structures/GroupView.js

```diff
@@ -935,7 +967,12 @@ export default React.createClass({
                             { rightButtons }
                         </div>
                     </div>
-                    { roomBody }
+                    <ScrollPanel className="mx_GroupView_body"
```

Comment:
ScrollPanel might be overkill here as it's designed for timeline style infinite pagination things - in UserSettings we just put a GeminiScrollbar in, for instance.

Reply:
Ah, that would make sense